internal/auth/jwt.go

```diff
@@ -50,7 +50,6 @@ func NewJWTMiddleware(issuer, jwksURI, contractAddress string, logger *zerolog.L
 		jwtValidator.ValidateToken,
 		jwtmiddleware.WithCredentialsOptional(false),
```

Comment:
I'm pretty sure this is what we want.

In the end I don't think it's affecting security, since everything is checking for a token id anyway.

Reply:
We need this to allow for GQL API discovery. Without this option, you need a valid JWT to view the documents.

Reply:
Then we delete the other line?